tests/Dialect/Lattigo/Emitters/emit_lattigo.mlir

```diff
@@ -197,3 +197,26 @@ module attributes {scheme.bgv} {
     return %ct1 : !lattigo.rlwe.ciphertext
   }
 }
+
+// -----
+
+// func test_negate_new(evaluator *bgv.Evaluator, ct *rlwe.Ciphertext) (*rlwe.Ciphertext) {
```

Comment:
why is this commented out?

Reply:
I pasted it for github co-pilot to complement FileCheck directive :)

Also as the FileCheck directive below is too complex I left this original version. But as jeremy pointed out in another PR these comments can become stale, they are deleted now.